rules/rbe_repo/container.bzl

```diff
@@ -24,14 +24,39 @@ load(
 # https://github.com/bazelbuild/bazel/issues/1262
 _EXTERNAL_FOLDER_PREFIX = "external/"
 
-_ROOT_DIR = "/rbe_autoconf"
+_BAZELISK_RELEASE = "v1.3.0"
```

Comment:
A general comment:

Suggest adding a file like `os_family.bzl` that takes a `ctx` object and returns a string describing the environment. Today it would return `Linux` and `Windows`. For environment specific variables like `_BAZELISK_FILENAME` etc, I would suggest creating a central dict thay maps from values returned by the function in `os_family.bzl` to environment specific values like:
```
_BAZELISK = {
  "Linux": {
     "FILENAME": "bazelisk",
     "SHA256": ...
  },
  "Windows": {
     ...
  }
}
```

This will avoid have long if-else environment specific ladders throughout the code and easier to add environments in the future. Also, I suggest looking into using [structs](https://docs.bazel.build/versions/master/skylark/lib/struct.html) for the inner objects in my code snippet above instead of nested dictionaries.